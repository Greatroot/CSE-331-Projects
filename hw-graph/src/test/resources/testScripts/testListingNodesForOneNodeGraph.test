# Tests listing nodes for one node graph

CreateGraph graph
AddNode graph n1
ListNodes graph