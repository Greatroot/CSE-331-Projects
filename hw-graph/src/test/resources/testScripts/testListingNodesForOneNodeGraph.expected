# Tests listing nodes for one node graph

created graph graph
added node n1 to graph
graph contains: n1